backend/package.json

```diff
@@ -80,6 +80,6 @@
     "graphql-upload": "^11.0.0"
   },
   "engines": {
-    "node": "14.15.5"
+    "node": "*"
```

Comment:
I'm not sure this is a change we want to make. I know it can sometimes cause difficulty and throw errors when it isn't a big deal, but I don't think we should commit this change. Could you revert this?

Reply:
Yep for sure, I think this change accidentally made it into the commit. I initially changed this because I was running into some node compatibility issues earlier .